Promact.Oauth.Server/src/Promact.Oauth.Server/Services/MessageServices.cs

```diff
@@ -46,18 +45,17 @@ public Task SendEmailAsync(string email, string subject, string message)
             {
                 using (var smtp = new SmtpClient())
                 {
-                    _logger.LogInformation("Start Email Sending");
+                  
                     smtp.ConnectAsync(_appSettings.Value.Host, _appSettings.Value.Port, MailKit.Security.SecureSocketOptions.None).Wait();
                     smtp.AuthenticateAsync(credentials: new NetworkCredential(_appSettings.Value.UserName, _appSettings.Value.Password)).Wait();
-                    smtp.SendAsync(msg, CancellationToken.None).Wait();
-                    smtp.DisconnectAsync(true, CancellationToken.None).Wait();
+                    smtp.SendAsync(msg).Wait();
```

Comment:
Calling an async method and calling wait does not make sense here. either keep Async properly or use synchronous method.